.github/ISSUE_TEMPLATE/Bug.md

```diff
@@ -0,0 +1,17 @@
+---
+name: 🐛 Bug
+about: Issues for filing a Bug
+labels: bug
```

Comment:
Just a question, should we add the `type: bug` label here to reference our existing labels? or he had just created a new label.

Reply:
This is the GH syntax for picking up existing labels. 

You can check them out actually working on my local repo at https://github.com/bryceosterhaus/clay/issues/new/choose